### PR TITLE
updated "Observability" to spanish

### LINKED ISSUE #3023: [es] Update "Observability" term in Spanish
Update the `content/es/observability.md` term since it is outdated with the original one in English.

### PR DESCRIPTION
**Pull request by SullyMA**
### Checklist before opening this PR (put `x` in the checkboxes)
- [✔️] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [ ✔️] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

**Comment by seokho-son** [comment]
Hi @SullyMA 
Please update this PR to remain related commits only. You don't have to include the other commits. 
(usually, it is because your feature branch is based on main branch not dev-es branch) 
:)

**Review by ramrodo — APPROVED** [review]
Thanks, @SullyMA. LGTM.

**Comment by ramrodo** [comment]
@SullyMA could you add to the PR's description the related issue, please?

resolves https://github.com/cncf/glossary/issues/3023

File: content/es/observability.md [hunk 1]
```diff
@@ -5,18 +5,12 @@ category: Concepto
...
 
-La observabilidad de un sistema tiene un impacto significativo en su costo operativo.
-Los sistemas observables brindan datos significativos y procesables a sus operadores, lo que les permite lograr resultados favorables (respuesta más rápida a incidentes, mayor productividad del desarrollador) y menos trabajo y tiempo de inactividad.
-
-## ¿Cómo ayuda?
-
-Es esencial entender que más información no necesariamente significa que en un sistema sea más observable.
-De hecho, a veces, la cantidad de información generada por un sistema puede dificultar la identificación de señales de salud valiosas debido al ruido generado por la aplicación.
-La observabilidad requiere los datos correctos en el momento correcto para que el consumidor correcto (humano o pieza de software) tome las decisiones correctas.
+Consecuentemente, cuán observable es un sistema afectará significativamente sus costos operativos y de desarrollo.
```
Comment (electrocucaracha) [inline content/es/observability.md:16]:
```suggestion
Por consiguiente, cuán observable sea un sistema esto afectará significativamente sus costos operativos y de desarrollo.
```

**Comment by SullyMA** [comment]
> @SullyMA could you add to the PR's description the related issue, please?
> 
> resolves #3023

**Review by electrocucaracha — APPROVED** [review]

**Pull request merged**